contrib/opencensus-ext-flask/README.rst

```diff
@@ -0,0 +1,31 @@
+OpenCensus Flask Integration
+============================================================================
+
+Installation
+------------
+
+::
+
+    pip install opencensus-ext-flask
+
+Usage
+-----
+
+.. code:: python
+
+    from flask import Flask
+    from opencensus.ext.flask.flask_middleware import FlaskMiddleware
+    from opencensus.trace.propagation.trace_context_http_header_format import TraceContextPropagator
+    
+    app = Flask(__name__)
+    middleware = FlaskMiddleware(app, propagator = TraceContextPropagator(), blacklist_paths = ['_ah/health'])
```

Comment:
Nit: spaces around `=`

Reply:
Shall we keep the spaces around `=` in the documentation? (Personally I would prefer to have spaces in docs for readability).
When I looked at https://github.com/census-instrumentation/opencensus-python, it seems most parts have spaces around the assignment operator. I also checked the standard Python Tutorial, and found spaces around `=` as well.

Reply:
It seems the recommended rule is to put space around assignment operator, but remove spaces around named arguments. I will use this approach, and we'll need to cleanup the existing docs. @c24t this sounds okay to you?

Reply:
[PEP8](https://www.python.org/dev/peps/pep-0008/#other-recommendations) and the [google style guide](http://google.github.io/styleguide/pyguide.html#36-whitespace) (which we're not following closely, admittedly) say not to do this. I think this hurts readability, but maybe just because I'm used to looking at code without the spaces.

Reply:
I don't think we're doing this elsewhere, `flake8` only turns up [E251 errors](https://lintlyci.github.io/Flake8Rules/rules/E251.html) in the generated proto code.

Reply:
(Sorry, replied before I saw your reply)

I think it's ok to break with the existing docs here and clean them up later.